src/Microsoft.ML.DataView/ImageDataViewType.cs

```diff
@@ -3,11 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Drawing;
-using Microsoft.ML.Data;
-using Microsoft.ML.Internal.Utilities;
-using Microsoft.ML.Runtime;
+using Microsoft.ML.Internal.DataView;
 
-namespace Microsoft.ML.Transforms.Image
+namespace Microsoft.ML.Data
 {
     public sealed class ImageDataViewType : StructuredDataViewType
```

Comment:
I thought the whole point of having the `ImageDataViewType` in a separate assembly is to show that the DataView type system is open and extensible. If we must have all the DataViewType class in `Microsoft.ML.DataView`, that completely breaks this extensibility story.

I'm not sure we should move ImageDataViewType into this assembly.

cc @tomfinley